src/System.Drawing.Common/tests/RegionTests.cs

```diff
@@ -0,0 +1,1930 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
You expect common sense of law!? 😃
 But, seriously, do you mean both the net foundation header and the mono one or just mono?